src/main/java/seedu/address/storage/JsonAdaptedInsurancePolicy.java

```diff
@@ -0,0 +1,42 @@
+package seedu.address.storage;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import seedu.address.model.insurancepolicy.InsurancePolicy;
+
+/**
+ * Jackson-friendly version of {@link InsurancePolicy}.
+ */
+class JsonAdaptedInsurancePolicy {
+
+    private final String policyId;
+
+    /**
+     * Constructs a {@code JsonAdaptedInsurancePolicy} with the given {@code policyId}.
+     */
+    @JsonCreator
+    public JsonAdaptedInsurancePolicy(String policyId) {
+        this.policyId = policyId;
+    }
+
+    /**
+     * Converts a given {@code Tag} into this class for Jackson use.
```

Comment:
I think this JavaDoc comment can be further refined.

Reply:
Have changed Tag into InsurancePolicy @ line 24. Will be keeping the JavaDoc comment to follow the JsonAdaptedTag class from which I adapted the code.